### ISSUE TITLE
CSS flexbox order property accepting pixel values

### ISSUE THREAD

**Issue by ManojAMK** [firebug/firebug#7922]
By definition, The `order` property for `flexbox` does not work with pixel values. But once it is edited through Firebug value spinner, it iterates through pixel values. Though the issue is not major and the  incorrect usage of pixel value is highlighted by the red box shadow in spinner, it is to be noted that currently `order` can accept negative values just as how `z-index` works.

![order](https://cloud.githubusercontent.com/assets/11471878/8768602/27f88c44-2ea4-11e5-8d71-29da10fcff9d.gif)

In below screenshot, I have used negative order values on the right hand side and it illustrates the corresponding output on the left.

![flexbox](https://cloud.githubusercontent.com/assets/11471878/8768664/12c471f6-2ea6-11e5-993a-f595341304b9.jpg)

In short, `order` spinner should works as `z-index` value spinner works.

**Comment by SebastianZ** [comment]
I've fixed that. Thanks for the report!

Sebastian

**Comment by ManojAMK** [comment]
Thanks for the fix, Sebastian! Cheers :+1:
